index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
```

Comment:
In this code too many of the tetris pieces are the same color, it might be beneficial to the game to vary the color more